src/Rule/ForbidCustomFunctionsRule.php

```diff
@@ -137,7 +137,7 @@ public function processNode(Node $node, Scope $scope): array
     }
 
     /**
-     * @return string[]
+     * @return list<string>
```

Comment:
FYI I also prefer this style but I found out it's not supported by PHPStorm by default -> it requires Psalm extension (and it's not described on the PHPStan webpage).